src/main/java/io/codelex/flightplanner/Flight.java

```diff
@@ -0,0 +1,93 @@
+package io.codelex.flightplanner;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@Valid
+public class Flight {
+    private String id;
+    @Valid
+    @NotNull
+    private Airport from;
+    @Valid
+    @NotNull
+    private Airport to;
+    @NotBlank
+    private String carrier;
+    @NotBlank
+
+    private String departureTime;
+    @NotBlank
+
+    private String arrivalTime;
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");

```

Comment:
Ja jau te ir tas formatter, varbūt tad uzreiz to izmanto iekš konstruktora un noparsē konsturktorā padotos String departureTime un String arrivalTime un pašā klasē glabā jau LocalDateTime, nevis String?